src/qoi/include/grins/absorption_coeff_testing.h

```diff
@@ -0,0 +1,63 @@
+//-----------------------------------------------------------------------bl-
+//--------------------------------------------------------------------------
+//
+// GRINS - General Reacting Incompressible Navier-Stokes
+//
+// Copyright (C) 2014-2016 Paul T. Bauman, Roy H. Stogner
+// Copyright (C) 2010-2013 The PECOS Development Team
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the Version 2.1 GNU Lesser General
+// Public License as published by the Free Software Foundation.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc. 51 Franklin Street, Fifth Floor,
+// Boston, MA  02110-1301  USA
+//
+//-----------------------------------------------------------------------el-
+
+
+#ifndef GRINS_ABSORPTION_COEFF_TESTING_H
+#define GRINS_ABSORPTION_COEFF_TESTING_H
+
+// GRINS
+#include "grins/absorption_coeff.h"
+
+namespace GRINSTesting
+{
+  class SpectroscopicAbsorptionTest;
+}
+
+namespace GRINS
```

Comment:
I would keep this in the `GRINSTesting` namespace

Reply:
So move it to `test/unit/` then as well?

Reply:
Yes, please.

Reply:
In fact, you ought not need a separate file, and just park that in place of `#include "absorption_coeff_testing.h"`

Reply:
Or keep it in `src/qoi/`?